#VRML_SIM R2021b utf8
WorldInfo {
  coordinateSystem "NUE"
}
Viewpoint {
  orientation -0.009173526666620345 -0.7449636905103938 -0.6670419373844729 3.130232670854303
  position 0.21803255216684653 12.801894420449644 -1.6128193506973847
  follow "e-puck:lidar_r"
  followType "None"
}
TexturedBackground {
}
TexturedBackgroundLight {
  castShadows FALSE
}
RectangleArena {
  floorSize 5 5
  wallThickness 0.2
  wallHeight 2
}
SolidBox {
  translation 1.5 0 1.6
  size 0.1 2 2.2
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -1.26339 0 1.49574
  name "box(4)"
  size 0.1 2 2
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -0.482132 0 1.00425
  name "box(12)"
  size 0.1 2 1.11
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -0.482132 0 1.00425
  name "box(13)"
  size 0.1 2 1.11
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation 1.5 0 -1.1
  name "box(6)"
  size 0.1 2 1.3
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation 0 0 -0.896753
  name "box(8)"
  size 0.1 2 1.3
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -1 -1.33227e-15 -2.07962
  name "box(10)"
  size 0.1 2 1.3
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -2.62355 0.09999999999999909 -1.5
  rotation 0 1 0 1.5708
  name "box(1)"
  size 0.1 2 1.75
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -0.871188 -8.88178e-16 -0.5
  rotation 0 1 0 1.5708
  name "box(11)"
  size 0.1 2 1.75
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -0.12243 -8.88178e-16 -1.48143
  rotation 0 1 0 1.5708
  name "box(9)"
  size 0.1 2 1.75
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation 1.87168 -8.88178e-16 -0.5
  rotation 0 1 0 1.5708
  name "box(7)"
  size 0.1 2 1.75
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation 0.538909 -8.88178e-16 0.505608
  rotation 0 1 0 1.5708
  name "box(5)"
  size 0.1 2 2
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation 0.521071 0 0.499913
  rotation 0 1 0 1.5708
  name "box(2)"
  size 0.1 2 2.05
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation 0 0 1.5
  rotation 0 1 0 1.5708
  name "box(3)"
  size 0.1 2 1
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -2.25 0 1.75
  rotation 0 1 0 1.5708
  name "box(14)"
  size 0.1 2 0.5
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
SolidBox {
  translation -1.50033 0 1.01995
  rotation 0 1 0 1.5708
  name "box(15)"
  size 0.1 2 0.5
  appearance PBRAppearance {
    baseColor 0 0 0
    baseColorMap ImageTexture {
      url [
        "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/tagged_wall.jpg"
      ]
    }
    roughness 0.5
    metalness 0
  }
  castShadows FALSE
}
E-puck {
  translation 0 -6.39543e-05 -2
  rotation 0 1 0 1.56999
  controller "e_puck_controller"
  supervisor TRUE
  groundSensorsSlot [
    DistanceSensor {
      translation -0.1 0 0
      rotation 0 1 0 1.5708003061004252
      name "distance sensor front"
      lookupTable [
        0 0 0
        0.1 1000 0
        0.2 2000 0
        0.3 3000 0
        0.4 4000 0
        0.5 5000 0
        1 10000 0
        2 20000 0
        3 30000 0
        5 50000 0
        10 100000 0
      ]
      numberOfRays 100
      aperture 1e-05
    }
    DistanceSensor {
      translation -0.1 0 0
      rotation -0.5773509358554485 0.5773509358554485 0.5773489358556708 -2.094395307179586
      name "distance sensor left"
      lookupTable [
        0 0 0
        0.1 1000 0
        0.2 2000 0
        0.3 3000 0
        0.4 4000 0
        0.5 5000 0
        1 10000 0
        2 20000 0
        3 30000 0
        5 50000 0
        10 100000 0
      ]
      numberOfRays 100
      aperture 1e-05
    }
    DistanceSensor {
      translation -0.1 0 0
      rotation -0.5773509358554485 -0.5773509358554485 0.5773489358556708 2.0944
      name "distance sensor right"
      lookupTable [
        0 0 0
        0.1 1000 0
        0.2 2000 0
        0.3 3000 0
        0.4 4000 0
        0.5 5000 0
        1 10000 0
        2 20000 0
        3 30000 0
        5 50000 0
        10 100000 0
      ]
      numberOfRays 100
      aperture 1e-05
    }
    DistanceSensor {
      translation -0.1 0 0
      rotation -0.6785997877224834 -0.2810849120718748 0.6785967877234219 2.59357
      name "distance sensor rightfront"
      lookupTable [
        0 0 0
        0.1 1000 0
        0.2 2000 0
        0.3 3000 0
        0.4 4000 0
        0.5 5000 0
        1 10000 0
        2 20000 0
        3 30000 0
        5 50000 0
        10 100000 0
      ]
      numberOfRays 100
      aperture 1e-05
    }
    DistanceSensor {
      translation -0.1 0 0
      rotation -0.6786002482181049 0.28108510281518717 0.6785962482166418 -2.593565307179586
      name "distance sensor leftfront"
      lookupTable [
        0 0 0
        0.1 1000 0
        0.2 2000 0
        0.3 3000 0
        0.4 4000 0
        0.5 5000 0
        1 10000 0
        2 20000 0
        3 30000 0
        5 50000 0
        10 100000 0
      ]
      numberOfRays 100
      aperture 1e-05
    }
    Lidar {
      rotation 1 0 0 1.57
      name "lidar_l"
      tiltAngle 1
      fieldOfView 0.785
      verticalFieldOfView 0.1
      numberOfLayers 1
      maxRange 8
    }
    Lidar {
      rotation -0.5773502691896258 -0.5773502691896258 -0.5773502691896258 0
      name "lidar_f"
      fieldOfView 0.785
      numberOfLayers 1
      maxRange 10
    }
    Lidar {
      rotation 1.9352105843330177e-07 0.7062102132387804 0.7080022137798715 3.13801
      name "lidar_r"
      fieldOfView 0.785
      verticalFieldOfView 0.1
      numberOfLayers 1
      maxRange 10
    }
  ]
}
E-puck {
  translation 2.0221 -6.39543e-05 0.0891368
  rotation 0 1 0 1.83179
  name "e-puck(1)"
  supervisor TRUE
}
E-puck {
  translation -1.86398 -6.39543e-05 0.176686
  rotation 0 1 0 -0.2626053071795864
  name "e-puck(2)"
  supervisor TRUE
}
E-puck {
  translation 1.98329 -6.39543e-05 -1.50212
  rotation 0 1 0 2.61719
  name "e-puck(3)"
  supervisor TRUE
}
E-puck {
  translation -1.52327 -6.39543e-05 -1.34337
  rotation 0 -1 0 -1.0463953071795862
  name "e-puck(4)"
  supervisor TRUE
}
E-puck {
  translation 1.0865 -6.43148e-05 1.37803
  rotation 0.0006286692353032981 -0.999999374287682 0.0009253103463324814 0.680072
  name "e-puck(5)"
  supervisor TRUE
}
E-puck {
  translation -1.80301 -6.43148e-05 1.8642
  rotation -0.00016239693909160047 0.9999996249413504 0.0008507316809256049 0.890725
  name "e-puck(6)"
  supervisor TRUE
}
E-puck {
  translation 0.347659 -6.43148e-05 -1.12723
  rotation -0.00016239693909160047 0.9999996249413504 0.0008507316809256049 0.890725
  name "e-puck(7)"
  supervisor TRUE
}
